docs/docs/developers/reference/environment_reference/cli_wallet_reference.md

```diff
@@ -41,6 +41,64 @@ You can create arbitrary aliases with the `alias` command. For example `aztec-wa
 
 :::
 
+## Paying Fees
+
+
+import { Why_Fees, CLI_Fees } from '/components/snippets';
+
+<Why_Fees />
+
+Below are all the payment methods available to pay transaction fees on Aztec, starting with the simplest.
+
+### Sponsored Fee Paying Contract
+
+This is a special type of FPC (explained [below](#fee-paying-contract)) that can be used to pay for account deployment and regular txs.
+Note: the contract alias below is that of a sponsored fee paying contract.
```

Comment:
can we make our examples complete
this shows how to create Payment method with sponsoredFPC but not how to use it int he wallet. add the next line:
```
aztec-wallet create-account -a 0xA $PAYMENT_METHOD
```

also don't use alias. Our examples should be complete

Reply:
This was the only code snippet (referencing from source so it doesn't go stale) will replace with commands in doc.